### ISSUE TITLE
Unlabelled data Training

### ISSUE THREAD

**Issue by jaideep11061982** [AlibabaResearch/efficientteacher#80]
Thanks for wonder for repo
_train SSOD model from scratch, the default setting is to first conduct a 220-epoch burn-in training, and then enter the SSOD training, which has been introduced in our paper._

Could you help understand the above line @alibaba-oss 

1_ Also brief steps to follow to label unlabelled data. 
2) Which model to use to label unlabelled data at the end of training

3) What is purpose of "https://github.com/AlibabaResearch/efficientteacher/blob/main/configs/ssod/custom/yolov5l_transfer_ssod.yaml"  which one to use for custom data

4) IS the labelled data is fraction of total data which we label and rest we label through the efficient teacher method

5) modify the train: data/custom_train.txt in yolov5l_custom.yaml, and then enter the following script.
export CUDA_VISIBLE_DEVICES="0,1,2,3,4,5,6,7"
python -m torch.distributed.launch --nproc_per_node 8 --master_addr 127.0.0.2 --master_port 29502 train

Supervised training step is to train on how much amount of labelled data ? what to modify in this step of supervised ?

**Comment by jaideep11061982** [comment]
@alibaba-oss @meton-robean @kdy1999
